### ISSUE TITLE
gnomeTheme.tabsAsHeaderbar=true flag results in some issues

### ISSUE THREAD

**Issue by unaaagy** [rafaelmardojai/firefox-gnome-theme#761]
**Issue number 1**
The padding on the top and bottom of the tab are uneven when firefox is not maximized.
![Screenshot from 2024-03-08 19-07-25](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/155830121/97c552ca-af06-44f5-9e00-81263715875d)

This issue is resolved when the `firefox view` icon and the `new tab` icon are not present in the tab bar. 
![Screenshot from 2024-03-08 19-09-45](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/155830121/d7345de9-b3d8-45d3-9254-b6ad64a83b1c)

**Issue number 2**
When firefox is maximized and the `firefox view` icon and the `new tab` icon are not present in the tab bar, the tabs shift downwards.

![Screenshot from 2024-03-08 19-10-00](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/155830121/279b7f10-c3ba-44af-9214-b22cf81eb9f3)

**Setup information**
 - OS: Fedora 39
 - Installation method: RPM
 - Theme version: master
 - Optional features enabled: gnomeTheme.tabsAsHeaderbar

**Comment by tschai-yim** [comment]
Seems the fix broke the padding on the newest version:
![image](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/37117573/d06b0e48-3c82-40f0-9342-c593a8b59992)
Running Firefox `124.0.2` and the following config:
```nix
"extensions.activeThemeID" = "firefox-compact-dark@mozilla.org";
"gnomeTheme.normalWidthTabs" = true;
"gnomeTheme.tabsAsHeaderbar" = true;

"toolkit.legacyUserProfileCustomizations.stylesheets" = true;
"browser.uidensity" = 0;
"svg.context-properties.content.enabled" = true;
"browser.theme.dark-private-windows" = false;
"widget.gtk.rounded-bottom-corners.enabled" = true;
```

**Comment by rafaelmardojai** [comment]
@tschai-yim hmm, that separator shouldn't be there. Do you have any extra styles?

**Comment by tschai-yim** [comment]
You are right, I also have the [`adw-gtk3-dark`](https://github.com/lassekongo83/adw-gtk3) GTK3 theme installed to make these apps look like GTK4. Removing the dconf entry `/org/gnome/desktop/interface/gtk-theme` fixes the issue. Excuse the noise.

This also makes other apps that respect the config like GIMP or VirtualBox look a bit older again, which is fine though. For others reading, you can still configure GTK3 themes through  `.config/gtk-3.0/gtk.css` and `.config/gtk-3.0/settings.ini` which work on some apps like LibreOffice but not on Firefox.

**Comment by elmagio** [comment]
I understand this isn't your bug @rafaelmardojai but do you know of a way to set a different GTK theme only for Firefox? I also use [adw-gtk3-dark](https://github.com/lassekongo83/adw-gtk3) for the same reason I use Firefox-GNOME-theme, to get a consistent Libadwaita look in every program, so if possible I'd like to find a way to fix this issue without reverting to default Adwaita as my GTK3 theme.

**Comment by unaaagy** [comment]
@elmagio https://www.linuxuprising.com/2019/10/how-to-use-different-gtk-3-theme-for.html

Unfortunately, it doesn't seem to be working for firefox with the css theme for me.

**Comment by elmagio** [comment]
> @elmagio https://www.linuxuprising.com/2019/10/how-to-use-different-gtk-3-theme-for.html
> 
> Unfortunately, it doesn't seem to be working for firefox with the css theme for me.

Thanks, I remembered I'm using the Firefox Flatpak so I set the environment variable in Flatseal and that works.

**Comment by tschai-yim** [comment]
It seems when only setting `/org/gnome/desktop/interface/gtk-theme` to `adw-gtk3-dark` the issue no longer exists. Meaning we can now use Adw GTK3 next to this theme without issues :+1:
